assets/content/static/navbar.html

```diff
@@ -70,7 +72,7 @@ <h6 class="text-primary mb-md-1">Handbook</h6>
                                         <h6 class="text-primary mb-md-1">ScholarX</h6>
                                     </div>
                                 </a>
-                                <a href="/engineering-team.html" class="media d-flex align-items-center">
+                                <a href="../../../engineering-team/" class="media d-flex align-items-center">
```

Comment:
This one also would work
```suggestion
                                <a href="/engineering-team" class="media d-flex align-items-center">
```